src/absil/ilreflect.fs

```diff
@@ -685,10 +707,17 @@ let queryableTypeGetMethod cenv emEnv parentT (mref:ILMethodRef) =
         let cconv = (if stat then BindingFlags.Static else BindingFlags.Instance)
         let methInfo = 
             try 
+#if FX_RESHAPED_REFLECTION
+              parentT.GetMethod(mref.Name,cconv ||| BindingFlags.Public ||| BindingFlags.NonPublic,
+                                null,
+                                argTs,
+                                (null:obj[]))
+#else
               parentT.GetMethod(mref.Name,cconv ||| BindingFlags.Public ||| BindingFlags.NonPublic,
                                 null,
                                 argTs,
-                                (null:ParameterModifier[])) 
+                                (null:ParameterModifier[]))
+#endif
```

Comment:
I run hot and cold on the readability of #ifdefing individual arguments, rather than the entire method declaration.  But for sure I have ifdef'd individual arguments a bunch of times, so I will change it here too.